website/content/docs/connect/proxies/envoy.mdx

```diff
@@ -253,6 +253,15 @@ defaults that are inherited by all services.
   specified, inherits the Envoy default for route timeouts (15s). A value of 0 will
   disable request timeouts.
 
+- `balance_inbound_connections` - The strategy used for balancing inbound connections
+  across Envoy worker threads. Connect's Envoy integration currently supports the
+  following `balance_inbound_connections` values:
+
+  - `""` - The default (empty) value. No connection balancing strategy will be used. 
+  - `exact_balance` - If set to this value, inbound connections to the service will
+    use the
+  [Envoy Exact Balance Strategy.](https://cloudnative.to/envoy/api-v3/config/listener/v3/listener.proto.html#config-listener-v3-listener-connectionbalanceconfig-exactbalance)
```

Comment:
```suggestion
  - `""` - Empty string (default). No connection balancing strategy is used. Consul does not balance inbound connections. 
  - `exact_balance` - Inbound connections to the service use the
  [Envoy Exact Balance Strategy.](https://cloudnative.to/envoy/api-v3/config/listener/v3/listener.proto.html#config-listener-v3-listener-connectionbalanceconfig-exactbalance)
```
Use present tense instead of future tense per the style guide. I added a little more information about what happens when the default value is set, but I'm not sure if this is accurate. Please verify.